src/Nethermind/Nethermind.Db.Rocks/DbOnTheRocks.cs

```diff
@@ -591,7 +593,10 @@ internal Span<byte> GetSpanWithColumnFamily(ReadOnlySpan<byte> key, ColumnFamily
         {
             Span<byte> span = _db.GetSpan(key, cf);
             if (!span.IsNullOrEmpty())
+            {
+                Interlocked.Increment(ref _allocatedSpan);
```

Comment:
Do we want to keep that in the prod (non-test) code?

Reply:
Ah, I suppose we should.